lib/reply.js

```diff
@@ -203,7 +208,7 @@ function handleError (reply, err, cb) {
 
   cb(reply, Object.assign({
     error: statusCodes[reply.res.statusCode + ''],
-    message: err.message,
+    message: (err || undefined) && err.message,
```

Comment:
Maybe I'm missing something here, why `err || undefined`?

Reply:
Since `err` can be any value this ensures any falsy evaluation of `err` sets the `message` property to `undefined`. That way it will not be output into the JSON response. Otherwise the falsy value of `err` (e.g. `null`, `false`, `0`, `''`) would be sent.

Reply:
I think the `message` property should be always there.
If you don't have a value to write, just use an empty string.

Reply:
@delvedor I've added a few extra test cases to demonstrate the behaviour in 6a69afa. Notice that it expects the `message` property of the response to not exist.

Reply:
@delvedor Sorry, didn't see your second comment in time. I've updated (34ffb0d) to output an empty string as the default error message.